misc/trustee-operator-install/install.sh

```diff
@@ -5,6 +5,7 @@ OCP_PULL_SECRET_LOCATION="${OCP_PULL_SECRET_LOCATION:-$HOME/pull-secret.json}"
 MIRRORING=false
 ADD_IMAGE_PULL_SECRET=false
 GA_RELEASE=true
+TRUSTEE_IMAGE=quay.io/niteesh/kbs-snp-vcek:v0.10.1
```

Comment:
Do you have the Dockerfile to build this? We can put the custom image under https://quay.io/openshift_sandboxed_containers/

Reply:
eg
`TRUSTEE_IMAGE=${TRUSTEE_IMAGE:-quay.io/openshift_sandboxed_containers/kbs:v0.10.1`

Also will the same image work for both SNP and TDX ?